SUPPORTED.md

```diff
@@ -0,0 +1,116 @@
+# Supported devices
+
+**If your device is unlisted but working, please open a pull request to update the list and add a fixture file (use `python -m devtools.dump_devinfo` to generate one).**
+
+## Tested device versions
+
+&#x25B8;&nbsp; Expand to see tested versions<br>
+<sup>*</sup>&nbsp; Model requires authentication<br>
+<sup>**</sup> Newer versions require authentication
+
+<!--SUPPORTED_START-->
+### Supported Kasa devices
+
+**Plugs**
+
+<details><summary>EP10</summary>Hardware 1.0(US) Firmare 1.0.2</details>
+<details><summary>EP25<sup> *</sup></summary>Hardware 2.6(US) Firmare 1.0.1<sup> *</sup><br>Hardware 2.6(US) Firmare 1.0.2<sup> *</sup></details>
+<details><summary>ES20M</summary>Hardware 1.0(US) Firmare 1.0.8</details>
+<details><summary>HS100<sup> **</sup></summary>Hardware 2.0(US) Firmare 1.5.6<br>Hardware 4.1(UK) Firmare 1.1.0<sup> *</sup><br>Hardware 1.0(US) Firmare 1.2.5<br>Hardware 1.0(UK) Firmare 1.2.6</details>
```

Comment:
Given this is a separate from README.md and we are not aiming for minimalism, I think we should just use a regular list (`<ul><li>xxx</li></ul>`) to avoid the need to click around to see the details.

Reply:
Done, now looks like this:
![image](https://github.com/python-kasa/python-kasa/assets/51370195/6bcb176a-c795-4bf8-be5a-2b9c1e1a262b)